### PR TITLE
API-Rule v1beta1

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Create webhook service 
- Update API-Rule CRD
- Create certificate management job

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/9936

### CODE REVIEW

File: resources/api-gateway/templates/webhook-service.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "api-gateway.fullname" . }}-webhook-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+    - port: 443
```
[the remark this answers is not in the document]
Comment (barchw) [reply resources/api-gateway/templates/webhook-service.yaml:8]: Added name: https-convert

**Review by cnvergence — APPROVED** [review]

**Review by cnvergence — APPROVED** [review]

**Pull request merged**